.github/workflows/dispatch-matrix-check.yml

```diff
@@ -0,0 +1,36 @@
+name: 🤖 Run Matrix Check 
+
+on:
+  push:
+    branches:
+      - main
+      - "rc/**"
+      - next
```

Comment:
Did you mean to include the push triggers here? The client-payload below includes a `pr` entry, which presumably isn't applicable when pushing.

Reply:
Yes, thanks!